SignalServiceKit/src/Storage/TSDatabaseView.m

```diff
@@ -144,7 +144,7 @@ + (void)asyncRegisterUnseenDatabaseView
 
     [self registerMessageDatabaseViewWithName:TSUnseenDatabaseViewExtensionName
                                  viewGrouping:viewGrouping
-                                      version:@"1"
+                                      version:@"2"
```

Comment:
I don't think we need to bother with rebuilding the unread database view, since its just used for badge counts.